src/views/GpaCalculator/GpaCalculator.vue

```diff
@@ -38,42 +54,35 @@
         </div>
         <div class="course-settings-row">
           <dropdown
-            v-show="true"
             style="flex:1;"
             :options="['Regular','Accelerated','Honors/AP']"
             :value="course.level"
             align="left"
             @input="selectedCourseLevel(course,$event)"
           />
-          <div style="width:8px;" />
-          <dropdown
-            v-show="true"
-            style="flex:1;"
-            :options="['Has Final','No Final']"
-            :value="course.hasFinal ? 0 : 1"
-            align="left"
-            @input="selectHasFinal(course,$event)"
-          />
-        </div>
-        <div class="grade-dropdown-row">
-          <div style="flex:1;" v-for="n in (course.hasFinal ? 4 : 3)" :key="n">
-            <div>
-              <div class="term">{{terms[n-1]}}</div>
-              <dropdown
-                v-show="true"
-                :direction="course.direction"
-                class="grade-selector"
-                :options="gradeLabels"
-                :value="course.grades[n-1]"
-                align="left"
-                @input="selectGrade(course,n-1,$event)"
-              />
-            </div>
+            <dropdown
+              style="flex:1;"
+              :direction="course.direction"
+              class="grade-selector"
+              :options="gradeLabels"
+              :value="course.grade"
+              align="left"
+              @input="selectGrade(course,$event)"
+            />
           </div>
-        </div>
         <p class="grade-label">{{course.finalGrade}}</p>
-        <div class="final-gpa">{{course.gpa.toString().length == 1 ? ( course.gpa+".0") : course.gpa}}</div>
 
+          <div class="gpa-title-row">
+            <div class="gpa-col">
+              <p class="weight-title"><b>Un</b>weighted</p>
+              <div class="final-gpa">{{course.unweightedGPA.toString().length == 1 ? ( course.unweightedGPA+".0") : course.unweightedGPA}}</div>
```

Comment:
This formatting pops up a few times, there might be a way to instead just always round the GPA value to two (or three) which would introduce the extra 0s if needed.